### DISCUSSION TITLE
Claim vote Banner / Button

### DISCUSSION THREAD

**Discussion by Toneeno** [gameserverapp/Platform#444] [Suggestions]
You should add a better claim vote system.
Would be nice to have like a button to claim a vote that could be posted on the webpage wherever you want like content,
in this way users could create Claim Vote pages.

Could be better to add a vote button too (as webpage content) to create a dedicated page where users can vote and claim easy it right after.

**Comment by mvaessen** [comment]
Hi,

Upon further investigation did I came across some things that need to be changed before this suggestion can be implemented. At this moment claiming happens per server, so adding a claim button would require to select a server too. This means that for people with multiple servers, you need multiple buttons. 

A better solution has to be found to work with this, without causing too much load for voting sites when customers have 20 game servers (basically resulting in 100 requests in a matter of seconds).
